src/Rx/Weno/TransmitProperties.php

```diff
@@ -0,0 +1,213 @@
+                             'LastName' => $this->patient['lname'],
+                            'FirstName' => $this->patient['fname'],
+                           'MiddleName' => $this->patient['mname'],
+                               'Prefix' => 'NA',
+                               'Suffix' => 'NA',
+                               "Gender" => $gender[0],
+                          "DateOfBirth" => $this->patient['dob'],
+                         "AddressLine1" => $this->patient['street'],
+                         "AddressLine2" => "NA",
+                                 "City" => $this->patient['city'],
+                                "State" => $this->patient['state'],
+                           "PostalCode" => $this->patient['postal_code'],
+                          "CountryCode" => "US",
+                         "PrimaryPhone" => $this->patient['phone_cell'],
+                          "SupportsSMS" => "Y",
+                         "PatientEmail" => $this->patient['email'],
+                        "PatientHeight" => $this->vitals['height'],
+                        "PatientWeight" => $this->vitals['weight'],
+          "HeightWeightObservationDate" => $heighDate[0],
+        "ResponsiblePartySameAsPatient" => 'Y',
+                      "PatientLocation" => "Home",
+                 "PrimaryPharmacyNCPCP" => $this->ncpdp,
+             "AlternativePharmacyNCPCP" => $this->ncpdp
+        ];
+
+        return json_encode($wenObj);
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getProviderEmail()
+    {
+        $provider_info = sqlQuery("select email from users where username=? ", [$_SESSION["authUser"]]);
+        if (empty($provider_info['email'])) {
+            echo xlt('Provider email address is missing');
+            exit;
+        } else {
+            return $provider_info;
+        }
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getFacilityInfo()
+    {
+        $locid = sqlQuery("select name, street, city, state, postal_code, phone, fax, weno_id from facility where id = ?", [isset($_SESSION['facilityId'])]);
+
+        if (empty($locid['weno_id'])) {
+            //if not in an encounter then get the first facility location id as default
+            $default_facility = sqlQuery("select name, street, city, state, postal_code, phone, fax, weno_id from facility order by id asc limit 1");
+
+            if (empty($default_facility)) {
+                echo xlt('Facility ID is missing');
+                exit;
+            } else {
+                return $default_facility;
+            }
+        }
+        return $locid;
+    }
+
+    /**
+     * @return mixed
+     */
+    private function getPatientInfo()
+    {
+        //get patient data
+        $patient = sqlQuery("select title, fname, lname, mname, street, state, city, email, phone_cell, postal_code, dob, sex, pid from patient_data where pid = ?", [isset($_SESSION['pid'])]);
```

Comment:
same issue as above with the isset() call

Reply:
done